tmcprototype/configure_hdbpp.py

```diff
@@ -0,0 +1,201 @@
+
+            except Exception as except_occured:
+                print("except_occured: ", except_occured)
+                ret_value = False
+                break
+    return ret_value
+
+
+# with open("/attribute_fqdn.txt") as json_file:
+#     attribute_list = json.load(json_file)
+#     for attribute in attribute_list:
+#         print("attribute: ", attribute)
+#         print("type",type(attribute))
+#         try:
+#             print("AttributeAdd on evt subscriber--------------------------------------------------------")
+#             evt_subscriber_proxy.command_inout("AttributeAdd",[attribute, "ALWAYS", "0"])
+#             #evt_subscriber_proxy.AttributeAdd([attribute, "ALWAYS", "0"])
+#         except Exception as except_occured:
+#             print("except_occured: ", except_occured)
+#
+#             pass
+
+
+# def cm_configure_attributes():
+#     ret_value = False
+#     try:
+#         # SetAttributeName
+#         conf_manager_proxy.write_attribute("SetAttributeName",
+#                                            "tango://archiver-1234-databaseds:10000/sys/tg_test/1/ampli")
+#         print("SetAttributeName: ", conf_manager_proxy.SetAttributeName)
+#
+#         # SetArchiver
+#         conf_manager_proxy.write_attribute("SetArchiver",
+#                                            "tango://archiver-1234-databaseds:10000/archiving/hdbpp/eventsubscriber01")
+#         print("SetArchiver: ", conf_manager_proxy.SetArchiver)
+#
+#         # SetStrategy
+#         conf_manager_proxy.write_attribute("SetStrategy", "ALWAYS")
+#         print("SetStrategy: ", conf_manager_proxy.SetStrategy)
+#
+#         # SetPollingPeriod
+#         conf_manager_proxy.write_attribute("SetPollingPeriod", 1000)
+#         print("SetPollingPeriod: ", conf_manager_proxy.SetPollingPeriod)
+#
+#         # SetEventPeriod
+#         conf_manager_proxy.write_attribute("SetPeriodEvent", 3000)
+#         print("SetPeriodEvent: ", conf_manager_proxy.SetPeriodEvent)
+#         ret_value = True
+#
+#     except Exception as except_occured:
+#         print("except_occured: ", except_occured)
+#         ret_value = False
+#     return ret_value
+
+def cm_add_attribute():
+    ret_value = False
+    try:
+        # Add Attribute
+        conf_manager_proxy.command_inout("AttributeAdd")
+        ret_value = True
+    except DevFailed as df:
+        # print("cm_add_attribute except_occured: ", df)
+        str_df = str(df)
+        print("cm_add_attribute except_occured reason: ", str_df)
+
+        if "reason = Already archived" in str_df:
+            start_archiving()
+            ret_value = True
+        else:
+            ret_value = False
+    return ret_value
+
+def start_archiving():
+    try:
+        conf_manager_proxy.command_inout("AttributeStart", "sys/tg_test/1/ampli")
+    except Exception as except_occured:
+        print("start_archiving except_occured: ", except_occured)
+
+# Main entrypoint of the script.
+if __name__ == "__main__":
```

Comment:
Please verify the script

Reply:
Updated script is present in the commit: [30a292232f386762e581837c55dd48d05be764e6](https://github.com/ska-telescope/tmc-prototype/pull/48/commits/6df733f21eb40002515379a8d0f535039b34c846)